graphistry/arrow_uploader.py

```diff
@@ -151,25 +164,58 @@ def __init__(self,
         self.__edge_encodings = edge_encodings
         self.__metadata = metadata
         self.__certificate_validation = certificate_validation
+        self.__org_name = org_name
     
-    def login(self, username, password):
+    def login(self, username, password, org_name=None):
+        from .pygraphistry import PyGraphistry
+
         base_path = self.server_base_path
         out = requests.post(
             f'{base_path}/api-token-auth/',
             verify=self.certificate_validation,
-            json={'username': username, 'password': password})
+            json={'username': username, 'password': password, "org_name": org_name})
         json_response = None
         try:
             json_response = out.json()
+            # print(json_response)
+            logged_in_org_name = json_response.get('active_organization',{}).get('slug', None)
+            print("set org_name")
+            PyGraphistry.org_name(logged_in_org_name)
+            
             if not ('token' in json_response):
                 raise Exception(out.text)
+
+            if org_name and (logged_in_org_name != org_name):
+                raise Exception("Login Organization does not found")
+
         except Exception:
             logger.error('Error: %s', out, exc_info=True)
             raise Exception(out.text)
```

Comment:
For older graphistry servers, shouldn't we allow this? And for recent ones, are there legit errors to expect, like 403 or 500?

Reply:
as long as they do not pass the "org_name" when register(), the old server will return None to logged_in_server. 
But if they use pygraphisry new version to pass in "org_name", then we raise Exception to tell them.